node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodePatcher.java

```diff
@@ -144,6 +147,16 @@ private Node applyField(Node node, String name, Inspector value) {
                 return nodeWithPatchedReports(node, value);
             case "openStackId" :
                 return node.withOpenStackId(asString(value));
+            case "minDiskAvailableGb":
```

Comment:
Perhaps you should add the names from NodeResources here as alternatives so that we can make things more consistent over time?

Reply:
Added the NodeResources names here, but would create too much of a mess to add it under node creation as well. For `/nodes/v3` we should perhaps have a `resources` object?